keyboards/bschwind/key_ripper/info.json

```diff
@@ -0,0 +1,112 @@
+{
+    "manufacturer": "bschwind",
+    "keyboard_name": "key-ripper",
+    "maintainer": "bschwind",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP29", "GP16", "GP17", "GP18", "GP9", "GP10", "GP19", "GP11", "GP12", "GP13", "GP14", "GP20", "GP22", "GP23"],
+        "rows": ["GP26", "GP25", "GP27", "GP28", "GP15", "GP24"]
+    },
+    "processor": "RP2040",
+    "url": "https://github.com/bschwind/key-ripper",
+    "usb": {
+        "device_version": "0.2.0",
+        "pid": "0x0200",
+        "vid": "0xFEED"
```

Comment:
This will need to be changed for VIA to merge the PR on their repo.
https://www.caniusevia.com/docs/configuring_qmk#change-vendor_id-product_id

Reply:
I see, would a vendor ID of `0x16C0` and an arbitrary, non-conflicting product ID be okay?

Reply:
@fauxpark sorry for the bump, for hobbyist boards, are there particular vendor IDs people pick when defining a VIA config?

Reply:
@fauxpark I updated with a new PID which shouldn't clash with any existing product IDs in the `0xFEED` vendor ID. The `0xB501` string also doesn't occur anywhere in this repo except for some data bytes in a header file unrelated to vendor or product IDs.